aws/cluster/templates/cluster-spec.yaml

```diff
@@ -61,6 +62,11 @@ ${oidc-config}
     storageBackend: ${apiserver-storage-backend}
     runtimeConfig:
 ${apiserver-runtime-config}
+    featureGates:
```

Comment:
This should be supported by ${apiserver-runtime-config} or configured using some other property not hardcoded